tests/test_core.py

```diff
@@ -8714,6 +8714,16 @@ def test_pthread_dylink_entry_point(self, args):
     main = test_file('core/pthread/test_pthread_dylink_entry_point.c')
     self.dylink_testf(main, need_reverse=False, emcc_args=args)
 
+  @needs_dylink
+  @node_pthreads
+  def test_pthread_dylink_exceptions(self):
+    self.emcc_args.append('-Wno-experimental')
+    self.set_setting('EXIT_RUNTIME')
+    self.set_setting('USE_PTHREADS')
+    self.emcc_args.append('-fexceptions')
+    main = test_file('core/pthread/test_pthread_dylink_exceptions.cpp')
+    self.dylink_testf(main, need_reverse=False)
```

Comment:
is the reverse a todo?

Reply:
I've been skipping the reverse testing for recently added "dylink + pthread" tests.. because it slows the tests down and makes them (a little) harder.   I can see that in this case it might be a good idea.   In general, I'm not sure we need to run our dynamic linking tests with main and side code flipped like that.   I'm not sure how much value or coverage it generates.

Reply:
BTW the version that landed does include the flipping